app/graphql/mutations/create_encounter.rb

```diff
@@ -21,9 +21,10 @@ def resolve(input)
                               user_name: input[:user_name])
 
     if encounter.save
-      encounter_monsters = input[:encounterMonsters]
-      encounter_monsters.each do |monster_name|
-        EncounterMonster.create!(encounter_id: encounter.id, monster_name: monster_name)
+      input[:encounterMonsters].each do |monster_index|
```

Comment:
this is just being slightly picky, we could probably change it from `encounterMonsters` to something easy to understand for frontend w/o reading the docs or us telling them. 

line 8: to `:encounterIndexes` or `encounterMonsterIndexes`
` argument :encounterMonsters, [String], required: true`

Reply:
Doesn't hurt to be as explicit as possible with variable nomenclature. I'll update this now.